api/base/urls.py

```diff
@@ -24,7 +24,8 @@
                 url(r'^docs/', include('rest_framework_swagger.urls')),
                 url(r'^institutions/', include('api.institutions.urls', namespace='institutions')),
                 url(r'^collections/', include('api.collections.urls', namespace='collections')),
-                url(r'^guids/', include('api.guids.urls', namespace='guids'))
+                url(r'^guids/', include('api.guids.urls', namespace='guids')),
+                url(r'^wikis/', include('api.wikis.urls', namespace='wikis'))
```

Comment:
Totally minor, but feel free to include the `,` at the end of this line because python.